legend-engine-xt-json-runtime/src/test/resources/queries/firmTreeDefectsResultSubset.json

```diff
@@ -0,0 +1 @@
+{"builder":{"_type":"json"},"values":[{"defects":[{"id":"validLatitude","externalId":null,"message":"Constraint :[validLatitude] violated in the Class GeographicPosition","enforcementLevel":"Error","ruleType":"ClassConstraint","ruleDefinerPath":"test::firm::model::GeographicPosition","path":[{"propertyName":"employees","index":0},{"propertyName":"addresses","index":0},{"propertyName":"address","index":null},{"propertyName":"position","index":null}]}],"source":{"number":1,"record":"{\"name\":\"Firm A\",\"ranking\":1,\"employees\":[{\"firstName\":\"Firstname A\",\"lastName\":\"LastName A\",\"dateOfBirth\":\"1970-12-13\",\"isAlive\":false,\"heightInMeters\":1.73,\"addresses\":[{\"addressType\":\"Home\",\"address\":{\"firstLine\":\"firstLine A\",\"secondLine\":\"secondLine A\",\"city\":\"city A\",\"region\":\"region A\",\"country\":\"country A\",\"position\":{\"latitude\":142.6,\"longitude\":24.6}}}]},{\"firstName\":\"Firstname B\",\"lastName\":\"LastName B\",\"dateOfBirth\":\"1970-12-14\",\"isAlive\":true,\"heightInMeters\":1.23,\"addresses\":[{\"addressType\":\"Home\",\"address\":{\"firstLine\":\"firstLine B\",\"secondLine\":\"secondLine B\",\"city\":\"city B\",\"region\":\"region B\",\"country\":\"country B\",\"position\":{\"latitude\":14.36,\"longitude\":14.36}}},{\"addressType\":\"Holiday\",\"address\":{\"firstLine\":\"firstLine C\",\"secondLine\":\"secondLine C\",\"city\":\"city C\",\"region\":\"region C\",\"country\":\"country C\",\"position\":{\"latitude\":1.36,\"longitude\":1.36}}}]}],\"addresses\":[{\"addressType\":\"Headquarters\",\"address\":{\"firstLine\":\"firstLine D\",\"secondLine\":\"secondLine D\",\"city\":\"city D\",\"region\":\"region D\",\"country\":\"country D\",\"position\":{\"latitude\":24.36,\"longitude\":24.36}}}]}"},"value":{"name":"Firm A","ranking":1}},{"defects":[],"source":{"number":2,"record":"{\"name\":\"Firm B\",\"employees\":[],\"addresses\":[{\"addressType\":\"Headquarters\",\"address\":{\"firstLine\":\"firstLine E\",\"secondLine\":\"secondLine E\",\"city\":\"city E\",\"region\":\"region E\",\"country\":\"country E\",\"position\":{\"latitude\":0.123,\"longitude\":0.145}}},{\"addressType\":\"RegionalOffice\",\"address\":{\"firstLine\":\"firstLine F\",\"secondLine\":\"secondLine F\",\"city\":\"city F\",\"region\":\"region F\",\"country\":\"country F\",\"position\":{\"latitude\":11.05,\"longitude\":13.26}}}]}"},"value":{"name":"Firm B","ranking":null}}]}
```

Comment:
Can we store them in a formatted way. Can help when we use them to showcase features